pyproject.toml

```
@@ -1,6 +1,73 @@
+[project]
+name = 'stpipe'
+description = 'Framework for calibration pipeline software'
+readme = 'README.md'
+requires-python = '>=3.7'
+license = { file = 'LICENSE' }
+authors = [{ name = 'STScI' }]
```

```suggestion
authors = [{ name = 'STScI', email = 'help@stsci.edu' }]
```

There is no email, which is breaking the docs build